.github/workflows/build-nightly-release.yaml

```diff
@@ -0,0 +1,78 @@
+name: Build nightly Catalyst releases for TestPyPI
+
+on:
+  schedule:
+    - cron: "40 23 * * 1-5"
```

Comment:
Can you add a comment here. I am reading this at 40th second 23 minutes, every hour?

Reply:
I know it is supposed to be 23:40 5 days a week, but I am not the most knowledgeable in cron and the few sources I found say that the first digits are the seconds.

Reply:
https://www.netiq.com/documentation/cloud-manager-2-5/ncm-reference/data/bexyssf.html

Reply:
https://bradymholt.github.io/cron-expression-descriptor/ This one says you are good. :sweat_smile: I'll approve.

Reply:
I looked at the github docs for this, they don't have seconds: 
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
```suggestion
    # Run every weekday at 23:40
    - cron: "40 23 * * 1-5"
```

Reply:
[Review: add cron comment](https://github.com/PennyLaneAI/catalyst/pull/921/commits/8287c2314aec1382a9dab03415cb3e4f927aabc8)